packages/experimental/ExperimentalIcon/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/experimental-icon",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
NPM says last published is 0.0.4

https://www.npmjs.com/package/@fluentui-react-native/experimental-icon